src/rpc/util.h

```diff
@@ -329,18 +329,46 @@ struct RPCExamples {
     std::string ToDescriptionString() const;
 };
 
+class RPCHelpMan;
+class RPCContext
+{
+public:
+    const RPCHelpMan& m_helpman;
+    const JSONRPCRequest& m_request;
```

Comment:
I believe the errors appearing are due to one or both of these going out of scope before the function call happens.

Reply:
Fixed.